test/include.test.js

```diff
@@ -33,14 +35,14 @@ describe('include', function() {
           should.not.exist(owner);
         } else {
           should.exist(owner);
-          owner.id.should.eql(p.ownerId);
+          should.equal(owner.id, p.ownerId);
```

Comment:
it turned out that there was a bug in Cassandra connector.  The `should.equal` changes across the board have all been reverted.  Thanks @raymondfeng for detailed comments about `should.equal(id1, id2)` vs. `id1.should.eql(id2)`  We should use the latter.